pkgs/development/tools/build-managers/meson/boost-Do-not-add-system-paths-on-nix.patch

```diff
@@ -1,26 +1,9 @@
-From 536108b10271f2f42d41c7d9ddb4ce2ea1851f4f Mon Sep 17 00:00:00 2001
-From: =?UTF-8?q?Niklas=20Hamb=C3=BCchen?= <mail@nh2.me>
-Date: Sat, 17 Oct 2020 19:27:08 +0200
-Subject: [PATCH] boost: Do not add system paths on nix
-
----
- mesonbuild/dependencies/boost.py | 17 +----------------
- 1 file changed, 1 insertion(+), 16 deletions(-)
-
-diff --git a/mesonbuild/dependencies/boost.py b/mesonbuild/dependencies/boost.py
-index 907c0c275..ecaf11b18 100644
 --- a/mesonbuild/dependencies/boost.py
 +++ b/mesonbuild/dependencies/boost.py
-@@ -643,22 +643,7 @@ class BoostDependency(ExternalDependency):
-             roots += [x for x in candidates if x.name.lower().startswith('boost') and x.is_dir()]
+@@ -682,16 +682,7 @@ class BoostDependency(SystemDependency):
          else:
              tmp = []  # type: T.List[Path]
--
--            # Homebrew
```

Comment:
Note: removed in https://github.com/mesonbuild/meson/commit/3c3fa0a12c68da8966410ad9bacd1148dc8bd65d

Reply:
Yea, I removed the hombrew lines since those were already removed in the upstream.